media-types.md

```diff
@@ -3,6 +3,7 @@
 The following `mediaType` MIME types are used by the formats described here, and the resources they reference:
 
 - `application/vnd.oci.descriptor.v1+json`: [Content Descriptor](descriptor.md)
+- `application/vnd.oci.descriptor.foreign.v1+json`: [Content Descriptor](descriptor.md)
```

Comment:
@wking Types of objects are not detected. The type of an object comes from trusted descriptors or a metadata storage system.

We are not adding licensing information in the course of this PR.

Please keep the conversation focused.